<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>637</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>450</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>341</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item row="4" column="2">
          <widget class="QDoubleSpinBox" name="SpinBox_gainGreen">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="SpinBox_gainRaw">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>exposure</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="horizontalSlider_gainGreen">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="SpinBox_exposure">
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="singleStep">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>10000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="horizontalSlider_gainRed">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="SpinBox_gainRed">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>gain</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSlider_gamma">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="horizontalSlider_gainBlue">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSlider_gainRaw">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="SpinBox_gamma">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>gamma</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QDoubleSpinBox" name="SpinBox_gainBlue">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>B</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSlider_exposure">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>W/B Auto</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Expo Auto</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="comboBox_Exp_Auto"/>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBox_WB_Auto">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Format</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>321</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="label_offsety">
           <property name="text">
            <string>offsetY</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_height">
           <property name="text">
            <string>height</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSlider_width">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="spinBox_width">
           <property name="maximum">
            <number>999996</number>
           </property>
           <property name="value">
            <number>2592</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="spinBox_offsetY">
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_width">
           <property name="text">
            <string>width</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_offsetx">
           <property name="text">
            <string>offsetX</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBox_offsetX">
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBox_height">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>2048</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_format">
           <property name="text">
            <string>Format</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_Format">
           <property name="currentText">
            <string>BayerGB8</string>
           </property>
           <item>
            <property name="text">
             <string>BayerGB8</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSlider_height">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSlider_offsetX">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="horizontalSlider_offsetY">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_conf_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_Recording_time">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>159</red>
            <green>158</green>
            <blue>158</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>159</red>
            <green>158</green>
            <blue>158</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recording: 00h 00m 00s</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_conf_save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCaptureStart"/>
   <addaction name="separator"/>
   <addaction name="actionOneShot"/>
   <addaction name="separator"/>
   <addaction name="actionRecordingStart"/>
   <addaction name="separator"/>
   <addaction name="actionPngImageSave"/>
   <addaction name="separator"/>
   <addaction name="actionFit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCaptureStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/new/prefix1/res/Start.png</normaloff>
     <normalon>:/new/prefix1/res/Stop.png</normalon>:/new/prefix1/res/Start.png</iconset>
   </property>
   <property name="text">
    <string>CaptureStart</string>
   </property>
  </action>
  <action name="actionOneShot">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/new/prefix1/res/OneShot_ToF.png</normaloff>:/new/prefix1/res/OneShot_ToF.png</iconset>
   </property>
   <property name="text">
    <string>OneShot</string>
   </property>
  </action>
  <action name="actionRecordingStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/new/prefix1/res/Recording.png</normaloff>
     <normalon>:/new/prefix1/res/RecordingStop.png</normalon>:/new/prefix1/res/Recording.png</iconset>
   </property>
   <property name="text">
    <string>RecordingStart</string>
   </property>
  </action>
  <action name="actionPngImageSave">
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/new/prefix1/res/SavePng.png</normaloff>:/new/prefix1/res/SavePng.png</iconset>
   </property>
   <property name="text">
    <string>PngImageSave</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar.qrc">
     <normaloff>:/new/prefix1/res/Image_Fit.png</normaloff>:/new/prefix1/res/Image_Fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="toolbar.qrc"/>
 </resources>
 <connections/>
</ui>
